swift/ios-shoppe-demo/Model/Order.swift

```diff
@@ -16,10 +16,10 @@ class Order {
         items.filter { $0.title == productName }.first?.quantity += 1
     }
 
-    func cartOrderTotal() -> Decimal {
-        var cost = Decimal()
+    func cartOrderTotal() -> Double {
```

Comment:
This could also be rewritten as

```swift
    func cartOrderTotal() -> Double {
        return items.reduce(0, { $0 + $1.getTotal() })
    }
```

(I think)

Reply:
Thanks so much I took a moment to learn about reduce  "a method to produce a single value from the elements of an entire sequence".